### PR TITLE
Written blog porterlb-boosts-kubernetes-on-bare-metal-en.

### PR DESCRIPTION
**Pull request by Patrick-LuoYu**
Signed-off-by: Patrick-LuoYu <patrickluo@yunify.com>
/assign @FeynmanZhou @zheng1 @duanjiong

### CODE REVIEW

File: content/en/blog/porterlb-boosts-kubernetes-on-bare-metal.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
+---
+title: "PorterLB: Boosts Kubernetes on Bare Metal"
```
Comment (FeynmanZhou) [inline content/en/blog/porterlb-boosts-kubernetes-on-bare-metal.md:2]: The title looks too simple, it has no relationship with PorterLB' core characteristics.

File: content/en/blog/porterlb-boosts-kubernetes-on-bare-metal.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+linkTitle: "PorterLB: Boosts Kubernetes on Bare Metal"
+date: 2021-02-25
```
Comment (FeynmanZhou) [inline content/en/blog/porterlb-boosts-kubernetes-on-bare-metal.md:4]: Please add keywords and description for each blog.

File: content/en/blog/porterlb-boosts-kubernetes-on-bare-metal.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+### Simple architecture
+
+After PorterLB is installed in a Kubernetes cluster, a porter-manager Deployment that contains a porter-manager Pod is created. The porter-manager Pod implements the functionality of PorterLB for the entire Kubernetes cluster. For high availability, you can scale the porter-manager Deployment and assign multiple PorterLB replicas (porter-manager pods) to multiple cluster nodes. This simple architecture ensures that PorterLB can be easily managed and integrated with other systems.
+
+### Seamless integration with KubeSphere
+
+KubeSphere is a successful open-source container platform and provides enterprise-grade features to simplify development and operations, such as observability and troubleshooting, unified monitoring and logging, centralized storage and networking management, and easy-to-use CI/CD pipelines. KubeSphere adopts a plug-and-play architecture to integrate third-party applications. As a sub-project of KubeSphere, PorterLB fits well into the KubeSphere ecosystem and can be seamless integrated as a plugin with KubeSphere.
```
Comment (FeynmanZhou) [inline content/en/blog/porterlb-boosts-kubernetes-on-bare-metal.md:61]: `successful` is strange if you describe an open source product.

Comment (Patrick-LuoYu) [reply content/en/blog/porterlb-boosts-kubernetes-on-bare-metal.md]: Title changed and other issues fixed.

**Review by FeynmanZhou — APPROVED** [review]

**Pull request merged**